doc/source/index.rst

```diff
@@ -128,5 +128,7 @@ Contact
 * `Matrix <https://matrix.org>`__
 
   An open network for secure, decentralized communication. Please find the
-  ``#kiwi`` room via `Riot <https://riot.im/app/>`__ on the web or by using
-  the supported `clients <https://matrix.org/docs/projects/clients-matrix>`__.
+  ``#kiwi`` room via
```

Comment:
`#kiwi:matrix.org` is the name of the room.

Reply:
updated